spec/lib/vmdb/config_spec.rb

```diff
@@ -3,10 +3,11 @@
   let(:enc_pass) { MiqPassword.encrypt(password) }
 
   it ".get_file" do
+    resource = FactoryGirl.create(:miq_server)
     stub_settings(:http_proxy => {:default => {:host => "proxy.example.com", :user => "user", :password => password, :port => 80}})
 
-    expect(VMDB::Config.get_file).to eq(
-      "---\n:http_proxy:\n  :default:\n    :host: proxy.example.com\n    :user: user\n    :password: #{enc_pass}\n    :port: 80\n"
+    expect(VMDB::Config.get_file(resource)).to eq(
+      "--- !ruby/object:Config::Options\ntable:\n  :http_proxy: !ruby/object:Config::Options\n    table:\n      :default: !ruby/object:Config::Options\n        table:\n          :host: proxy.example.com\n          :user: user\n          :password: #{enc_pass}\n          :port: 80\n        modifiable: true\n    modifiable: true\n"
```

Comment:
If that is the result of running your test, then something is broken, because we don't want yaml with Config::Options in it

Reply:
Try `.to_h.to_yaml` instead?

Reply:
@Fryguy fixed that, please re-review